docs/en/operation/Metrics-System.md

```diff
@@ -126,3 +126,85 @@ to the under store.
 
 Tags are additional pieces of metadata for the metric such as user name or under storage location.
 Tags can be used to further filter or aggregate on various characteristics.
+
+## Setting up Grafana for Alluxio
+
+Grafana is a metics analytics and visualization software used for visualizing time series
+data. You can use Grafana to better visualize the various metrics that Alluxio collects, and the software
+allows users to more easily see what's going on behind the scenes in Alluxio.
+
+Before setting up Grafana a monitoring tool must be set up that Grafana will pull metrics from. Alluxio
+supports exporting metrics to `Prometheus` and `Graphite`, so set up for both will be shown.
+
+### Setting up Prometheus
+
+Installing Prometheus using the instructions [here](https://prometheus.io/docs/prometheus/latest/installation/).
+
+After installing Prometheus, it must be configured to get metrics from Alluxio. First open the
+`prometheus.yml` file that Prometheus calls when starting. Next put the following sections under the
```

Comment:
"... file that Prometheus calls when starting."

- I think we can just remove this part, or just say something like "In the `prometheus.yml` configuration file add the following sections under.."